service/service_blackbox_test.go

```diff
@@ -0,0 +1,109 @@
+package service_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-common/errors"
+
+	testsupport "github.com/fabric8-services/fabric8-common/test"
+
+	clusterclient "github.com/fabric8-services/fabric8-cluster-client/cluster"
+	"github.com/fabric8-services/fabric8-cluster-client/service"
+	testauth "github.com/fabric8-services/fabric8-common/test/auth"
+	testsuite "github.com/fabric8-services/fabric8-common/test/suite"
+	"github.com/goadesign/goa/client"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	gock "gopkg.in/h2non/gock.v1"
+)
+
+var clusterURL = "https://cluster.prod-preview.openshift.io"
```

Comment:
we should avoid real URLs in tests, I believe. Unless you really intend to call the `cluster` service on preview?